articles/cognitive-services/Custom-Vision-Service/copy-move-projects.md

```diff
@@ -99,6 +99,9 @@ You'll get a `200/OK` response with metadata about the exported project and a re
 }
 ```
 
+> [!TIP]
+> While importing the project using the token generated by the export project step, if you face an error which says, "Invalid Token", it could be that the token url string is not web encoded. You can encode the token using the [URL Encoder](https://meyerweb.com/eric/tools/dencoder/)
```

Comment:
Rewrite:

If you get an "Invalid Token" error when you import your project, it could be that the token URL string isn't web encoded. You can encode the token using a [URL Encoder](https://meyerweb.com/eric/tools/dencoder/).

Reply:
@PatrickFarley Sure thing! Done!